### DISCUSSION TITLE
How do I connect to a self-hosted QFieldCloud instance from iOS

### DISCUSSION THREAD

**Discussion by ChipWolf** [opengisch/QField#5816] [Q&A]
As above

**Comment by suricactus** [comment]
Thank you for trying out QFieldCloud. Please check our documentation for more answers to your questions. This particular one can be found here: https://docs.qfield.org/get-started/tutorials/advanced-setup-qfc/#connect-to-a-custom-qfieldcloud-server-in-qfield-and-qfieldsync

**Reply by ChipWolf** [reply]
> Thank you for trying out QFieldCloud. Please check our documentation for more answers to your questions. This particular one can be found here: https://docs.qfield.org/get-started/tutorials/advanced-setup-qfc/#connect-to-a-custom-qfieldcloud-server-in-qfield-and-qfieldsync

There is no "Server URL" field on iOS

**Reply by nirvn** [reply]
@ChipWolf , did you follow the instructions on the documentation page to reveal the server url to begin with? Ie:

Double-tap on the Nyuki icon (the blue bee QFieldCloud logo). This action will reveal a field where you can enter the preferred QFieldCloud server address.

**Reply by ChipWolf** [reply]
> @ChipWolf , did you follow the instructions on the documentation page to reveal the server url to begin with? Ie:
> 
> 
> 
> Double-tap on the Nyuki icon (the blue bee QFieldCloud logo). This action will reveal a field where you can enter the preferred QFieldCloud server address.

I skimmed over that. I wish it was clearer in the UI, it doesn't feel as though it should be hidden.

Got there eventually though.

Thanks!
